src/inmanta/compiler/help/templates/modified_after_freeze.j2

```diff
@@ -20,9 +20,10 @@ This can mean one of two things:
 The procedure to solve this is the following:
 
 1. Ensure the model is correct by checking that the problematic assignment at {{location | underline}} is not conditional on the value it assigns.
-2. Report a bug to the inmanta issue tracker at https://github.com/inmanta/inmanta/issues or directly contact inmanta. This is a priority issue to us, so you will be helped rapidly and by reporting the problem, we can fix it properly.
-3. {%if reverse %}[applies]{%else%}[does not apply here]{%endif%} If the exception is on the reverse relation, try to give a hint by explicitly using the problematic relation{%if reverse_example%}: {{reverse_example | underline}}{%endif%}.
-4. Simplify the model by relying less on `is defined` but use a boolean instead.
+2. Set a relation precedence policy on the Inmanta project (See: https://docs.inmanta.com/community/latest/troubleshooting.html#compilation-fails).
```

Comment:
@wouterdb same question here. Up till now I understood that this was more something we could offer as an emergency resolution on a support request. Do we want the user to take this step themselves?

Reply:
I think so, it may make things easier for us. 

I think this is what @bartv  also preferred